packages/wallet/backend/src/walletAddress/service.ts

```diff
@@ -287,6 +289,11 @@ export class WalletAddressService implements IWalletAddressService {
     await WalletAddress.query().findById(id).patch({
       active: false
     })
+    console.log(walletAddress.accountId)
+    await this.transactionService.updateTransaction(
+      { accountId: walletAddress.accountId },
```

Comment:
it should be updated by wallet address id
there can be multiple wallet addresses in one account and it is not correct to mark as DELETED transactions that belongs to other wallet addresses that the one that is marked as inactive